### PR TITLE
feat(businessattribute): filter schema rows on business-attribute pro…

### PR DESCRIPTION
**Pull request by deepgarg-visa**
…perties

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts [hunk 1]
```diff
@@ -16,6 +16,24 @@ function matchesTagsOrTermsOrDescription(field: SchemaField, filterText: string,
     );
 }
 
+function matchesBusinessAttributesProperties(field: SchemaField, filterText: string, entityRegistry: EntityRegistry) {
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts:19]: nice! could we add some tests in `filterSchemaRows.test.ts` for this new functionality?

**Review by chriscollins3456 — APPROVED** [review]
looks good to me! I have a request to add tests but that can be done in a followup so this can get in

File: datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts (hunk 1 above)
Comment (deepgarg-visa) [reply datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts:19]: sure I will add that

**Review by chriscollins3456 — COMMENTED** [review]
it looks like `:datahub-web-react:yarnLint` is failing

**Pull request merged**